packages/m/moveable.json

```diff
@@ -0,0 +1,58 @@
+{
+  "description": "Moveable is Draggable, Resizable, Scalable, Rotatable, Warpable, Pinchable, Groupable, Snappable.",
+  "filename": "moveable.min.js",
+  "homepage": "https://daybrush.com/moveable",
+  "keywords": [
+    "moveable",
+    "resizable",
+    "scalable",
+    "draggable",
+    "rotatable",
+    "groupable",
+    "movable",
+    "warpable",
+    "pinchable",
+    "snappable",
+    "clippable",
+    "roundable",
+    "border",
+    "origin",
+    "dom",
+    "resize",
+    "scale",
+    "drag",
+    "move",
+    "warp",
+    "rotate",
+    "react",
+    "preact",
+    "throttle",
+    "group",
+    "pinchi",
+    "snap"
+  ],
+  "license": "MIT",
+  "name": "moveable",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/daybrush/moveable.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "moveable",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
```suggestion
          "*.js?(.map)"
```
Open wildcards are dangerous, let's just match what we need. [Glob test](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.js%3F%28.map%29&matches=false&tests=%2F%2F%20Imported%20from%20moveable%20NPM%20package&tests=.editorconfig&tests=LICENSE&tests=static%2Fscripts%2Fcustom.js&tests=dist%2Fmoveable.cjs.js&tests=dist%2Fmoveable.esm.js&tests=dist%2Fmoveable.js&tests=dist%2Fmoveable.min.js&tests=rollup.config.demo.js&tests=rollup.config.js&tests=jsdoc.json&tests=package.json&tests=tsconfig.declaration.json&tests=tsconfig.json&tests=tsconfig.test.json&tests=tslint.json&tests=dist%2Fmoveable.cjs.js.map&tests=dist%2Fmoveable.esm.js.map&tests=dist%2Fmoveable.js.map&tests=dist%2Fmoveable.min.js.map&tests=CHANGELOG.md&tests=custom_css.md&tests=groupable.md&tests=handbook%2Fhandbook.md&tests=.github%2FISSUE_TEMPLATE.md&tests=README.md&tests=declaration%2Fconsts.d.ts&tests=src%2Fconsts.ts&tests=declaration%2Findex.d.ts&tests=declaration%2Findex.esm.d.ts&tests=src%2Findex.esm.ts&tests=src%2Findex.ts&tests=declaration%2Findex.umd.d.ts&tests=src%2Findex.umd.ts&tests=declaration%2FInnerMoveable.d.ts&tests=declaration%2FMoveable.d.ts&tests=declaration%2Ftypes.d.ts&tests=src%2Ftypes.ts&tests=declaration%2Futils.d.ts&tests=src%2Futils.ts&tests=src%2FInnerMoveable.tsx&tests=src%2FMoveable.tsx&tests=.github%2FFUNDING.yml)